erpnext/buying/doctype/purchase_order/purchase_order.py

```diff
@@ -468,9 +468,10 @@ def update_item(obj, target, source_parent):
 
 @frappe.whitelist()
 def make_rm_stock_entry(purchase_order, rm_items):
+	rm_items_list = rm_items
 	if isinstance(rm_items, string_types):
```

Comment:
```suggestion
	if isinstance(rm_items, str):
```
`string_types` from `six` returns `str` only in py3. Let's remove it's usage wherever possible
![Screenshot 2021-06-08 at 1 17 42 AM](https://user-images.githubusercontent.com/25857446/121078611-cf602900-c7f6-11eb-82ce-0144b7b5a988.png)